packages/material-renderers/src/controls/MaterialDateControl.tsx

```diff
@@ -62,6 +67,8 @@ export const MaterialDateControl = (props: ControlProps) => {
     appliedUiSchemaOptions.showUnfocusedDescription
   );
 
+  const [key, setKey] = useState<any>(0);
```

Comment:
We should avoid `any` at all cost and only use it when there is no better way. Here we can just type as `number`.